src/features/projectsV2/ProjectsContext.tsx

```diff
@@ -188,15 +189,11 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
 
   useEffect(() => {
     async function loadProjects() {
-      if (page !== 'project-list' || !currencyCode) {
-        return;
-      }
-      if (projects !== null) {
-        return;
-      }
+      if (page !== 'project-list' || !currencyCode) return;
+      if (projectsLocale === locale && projects !== null) return;
+
```

Comment:
This is a false positive.

There is no difference between the suggested condition and the original condition, and we want to avoid calling the API if `projectsLocale === locale && projects !== null` , as we'd be fetching projects data again for the same language. We only fetch the data if the language has changed, or if projects were not fetched in the first place.